src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -88,12 +107,66 @@ public String getAdditionalSourceFileLocation() {
         return Arrays.asList(objectFile.toAbsolutePath().toString());
     }
 
+    @Override
+    public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguration) throws IOException, InterruptedException {
+        boolean result = super.link(paths, projectConfiguration);
+
+        if (result) {
+            createInfoPlist(paths, projectConfiguration);
+
+            if (! isSimulator()) {
+                CodeSigning codeSigning = new CodeSigning(paths, projectConfiguration);
+                if (! codeSigning.signApp()) {
+                    Logger.logSevere("Error signing the app");
+                }
+            }
+        }
+        return result;
+    }
+
     @Override
     public String getCompiler() {
         return "clang";
     }
 
+    @Override
+    String getAppPath(String appName) {
+        Path appPath = paths.getAppPath().resolve(appName + ".app");
+        if (!Files.exists(appPath)) {
+            try {
+                Files.createDirectories(appPath);
+            } catch (IOException e) {
+                Logger.logSevere("Error creating path " + appPath + ": " + e.getMessage());
```

Comment:
This seems fatal to me. Don't catch the exception.

Reply:
ok